src/languages/en.js

```diff
@@ -174,6 +174,9 @@ export default {
         beginningOfChatHistoryRestrictedPartOne: 'This is the beginning of ',
         beginningOfChatHistoryPrivatePartTwo: ' room, invite others by @mentioning them.',
         beginningOfChatHistoryRestrictedPartTwo: ', invite others by @mentioning them.',
+        beginningOfChatHistoryPolicyExpenseChatPartOne: 'Collaboration between ',
+        beginningOfChatHistoryPolicyExpenseChatPartTwo: ' and ',
+        beginningOfChatHistoryPolicyExpenseChatPartThree: ' starts here! 🎉 This is the place to chat, request money and settle up.',
```

Comment:
Do we have a way of adding macros to translations? Like this:

https://github.com/Expensify/App/blob/47b4541391d2b9e70e003b3d58e401bc9ee5e974/src/languages/en.js#L373

Or did that not feel like an option because of the bold text styling we needed to apply to the display names?

Reply:
Yeah I did it this way because of the bold styling. I was thinking there could be a way we could make a reusable "StyleText" component of some sort but I couldn't figure out how to generically make a template that does "bold" some words of a translation and not others. For now I figured this was easier.

Reply:
Yeah I can't think of a great way to do this. We probably don't need to optimize this yet - but good to think about for a second.